general/token/accesstokencreate.go

```diff
@@ -0,0 +1,150 @@
+package generic
```

Comment:
Let's rename this file to "createaccesstoken"

Reply:
I think we should follow the convention where the file is named after the command it adds.